apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathExpectationSamples.kt

```diff
@@ -252,10 +252,10 @@ class PathExpectationSamples {
             .notToEndWith("jpg")
 
         fails {
-            expect(dir).extension // subject is now of type String (actually "txt")
-                .toEqual("txtt") // fails because it doesn't equal to "txtt"
-                .toEndWith("jpg") // not reported
-            //                              use `.extension` if you want that all expectations are evaluated
+            expect(dir).extension //            subject is now of type String (actually "txt")
+                .toEqual("txtt") //     fails because it doesn't equal to "txtt"
+                .toEndWith("jpg") //    not reported
+            //                                  use `.extension` if you want that all expectations are evaluated
```

Comment:
That does not look well aligned. also, please align not only the text but also the `//` the only exception to that rule is if `//` starts on a new line (without prefixed code). The reason for this is that Intellij would move the // to the beginning of the indentation when formatting.
So please use the following approach:
```suggestion
            expect(dir).extension  // subject is now of type String (actually "txt")
                .toEqual("txtt")   // fails because it doesn't equal to "txtt"
                .toEndWith("jpg")  // not reported
            //                        use `.extension` if you want that all expectations are evaluated
```